Sources/Foundation/Date.swift

```diff
@@ -27,6 +27,13 @@ public struct Date : ReferenceConvertible, Comparable, Equatable {
         return CFAbsoluteTimeGetCurrent()
     }
     
+    /// Returns a `Date` initialized to the current date and time.
+    public static var now: Date {
+        get {
+            init()
+        }
+    }
+    
```

Comment:
This also works and is perhaps a bit clearer:

```
public static var now : Date { Date() }
```

Reply:
Thanks. Can only members run CI?

Reply:
Sorry I missed this comment from earlier -- yes, people with commit access can run CI.